docs/tutorials/tokens/erc1155.md

```diff
@@ -164,3 +164,64 @@ Other parameters:
 - You must specify a `pool` name if you've created more than one pool
 - You may specify a `key` understood by the connector (i.e. an Ethereum address) if you'd like to use a non-default signing identity
 - You may specify `from` if you'd like to burn tokens from a specific identity (default is the same as `key`)
+
+## Token approvals
+You can also approve other wallets to transfer tokens on your behalf with the `/approvals` API. The important fields in a token approval API request are as follows:
+
+- `approved`: Sets whether another account is allowed to transfer tokens out of this wallet or not. If not specified, will default to `false`.
+- `operator`: The other account that is allowed to transfer tokens out of the wallet specified in the `key` field
+- `config.amount`: The amount of tokens the other account is allowed to transfer. If `0` or not set, the approval is valid for any number. Required for `fungible` token pools.
+- `config.tokenIndex`: The specific token index within the pool that the operator is allowed to transfer. If `0` or not set, the approval is valid for any token. Required for `nonfungible` token pools.
```

Comment:
I don't think either of these config options is valid with the reference connector. Our ERC-1155 only supports "approval for all".

Reply:
Okay wasn't sure on that. I'll change it. Thanks